### DISCUSSION TITLE
cannot start up the pvc through the juicefs storage class

### DISCUSSION THREAD

**Discussion by starry-eyed-art** [juicedata/juicefs#2280] [Q&A]
error msg:
waiting for a volume to be created, either by external provisioner "csi.juicefs.com" or manually created by system administrator

juicefs csi has been installed like...
kube-system           juicefs-csi-controller-0                          3/3     Running            0          74m
kube-system           juicefs-csi-node-thcds                            3/3     Running            0          74m

sc has been started up like...
juicefs-sc             csi.juicefs.com         Retain          Immediate              false                  75m

**Comment by xiaogaozi** [comment]
Please follow the [troubleshooting guide](https://juicefs.com/docs/csi/troubleshooting) and provide more information.
